scripts/deployment/README.md

```diff
@@ -25,6 +25,14 @@ Parameters of the `globals.json` file:
 - `useLedger`: a flag whether to use the hardware wallet (`true`) or proceed with the seed-phrase accounts (`false`);
 - `derivationPath`: a string with the derivation path;
 - `providerName`: a network type (see `hardhat.config.js` for the network configurations);
+- `serviceStakingParams`: a set of service staking parameters, which include:
+  - `rewardsPerSecond`: amount of token (in wei) per second credited to a service if the service is active enough. Assuming the maximum
+    number of reward per day is 1 (in ETH), then the reward per second is calculated by
+    [this formula](https://www.wolframalpha.com/input?i=1+*+10%5E18+%2F+%283600+*+24+*+7%29);
+  - `livenessRatio`: number of service multisig transactions per second (in 1e18 value) that are used to measure the service
+    liveness (activity). In other words, it's the minimum number of transactions the service multisig needs to perform in order
+    to pass the liveness check. Assuming the number of required tx-s per day is 10, the liveness ratio is calculated by
+    [this formula](https://www.wolframalpha.com/input?i=%28115740740740740+*+60+*+60+*+24%29+%2F+10%5E18).
```

Comment:
```suggestion
    [this formula](https://www.wolframalpha.com/input?i=%28115740740740740+*+86400%29%2F10%5E18).
```

Reply:
I think the previous one was better for understanding - seconds, minutes, hours

Reply:
ok, then let's keep your :-)